caluma/caluma_form/serializers.py

```diff
@@ -240,12 +240,28 @@ class Meta(SaveQuestionSerializer.Meta):
 
 
 class SaveDateQuestionSerializer(SaveQuestionSerializer):
+    min_date = DateField("%Y-%m-%d", required=False, allow_null=True)
+    max_date = DateField("%Y-%m-%d", required=False, allow_null=True)
+
     def validate(self, data):
```

Comment:
@winged there is a validation in the backend, but since i finish this PR, it's waiting for review

Reply:
https://github.com/projectcaluma/ember-caluma/pull/2521

here is the implementation of the frontend side

Reply:
Right, but I think if we do this, we should add validation to the *answers* as well. This only validates the question, but no restriction happens when people actually fill out the forms.

Look at [the answer validator](https://github.com/projectcaluma/caluma/blob/1a47a86868a252951d3ec9e66ea94bc63f298f42/caluma/caluma_form/validators.py#L84C9-L84C32) to see what I mean.

There's already `_validate_question_float()` and `_validate_question_integer()` that implement the same pattern. I think therefore, you should extend `_validate_question_date()` to match.

Reply:
that's right, i will implement that too, and i will include it in my todo for this week